### DISCUSSION TITLE
v1.5-experimental2

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#4137] [Announcements]
## 1.5 Experimental 2 (1.5.240124002-experimental2) 🧪

The second 1.5 experimental release of Windows App SDK is now available! Releases in the experimental channel include features that are in the early stages of development. Experimental features may be removed from the next release, or may never be released. See [Windows App SDK release channels](https://docs.microsoft.com/windows/apps/windows-app-sdk/release-channels) for more information about the various release channels.

This release includes bug fixes from 1.4 and provides access to non-stable APIs and features for WinUI 3, System Backdrops, Content, Windowing, Input, and Controls.

**Check out the [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/experimental-channel#version-15-experimental-150-experimental2) for all of the new and updated features, limitations, and known issues.**

### Updating to this version

The 1.5.240124002-experimental2 package is available at: https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.5.240124002-experimental2

Additionally, for those testing deployment of unpackaged apps, download the Windows App SDK Installers and MSIX packages [here](https://aka.ms/windowsappsdk/1.5/1.5.231202003-experimental1/Microsoft.WindowsAppRuntime.Redist.1.5.zip).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.5-exp2'>v1.5-experimental2</a>.</em>

**Comment by Gaoyifei1011** [comment]
> ## 1.5 Experimental 2 (1.5.240124002-experimental2) 🧪
> The second 1.5 experimental release of Windows App SDK is now available! Releases in the experimental channel include features that are in the early stages of development. Experimental features may be removed from the next release, or may never be released. See [Windows App SDK release channels](https://docs.microsoft.com/windows/apps/windows-app-sdk/release-channels) for more information about the various release channels.
> 
> This release includes bug fixes from 1.4 and provides access to non-stable APIs and features for WinUI 3, System Backdrops, Content, Windowing, Input, and Controls.
> 
> **Check out the [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/experimental-channel#version-15-experimental-150-experimental2) for all of the new and updated features, limitations, and known issues.**
> 
> ### Updating to this version
> The 1.5.240124002-experimental2 package is available at: https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.5.240124002-experimental2
> 
> Additionally, for those testing deployment of unpackaged apps, download the Windows App SDK Installers and MSIX packages [here](https://aka.ms/windowsappsdk/1.5/1.5.231202003-experimental1/Microsoft.WindowsAppRuntime.Redist.1.5.zip).
> 
> _This discussion was created from the release [v1.5-experimental2](https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.5-exp2)._

In wasdk 1.5, the plan also includes ink workspace and segmented controls, will that come in the next release.

Segmented control are even available in the c# version of Community Toolkit 8.0

**Reply by ghost1372** [reply]
not sure
> Update: Although we are continuing to investigate the table view and ink controls, they aren't likely to fully land in the 1.5 timeline at this time.

https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md

**Reply by Gaoyifei1011** [reply]
😓😓😓

**Reply by duncanmacmichael** [reply]
@ghost1372 is correct - if by "next release" you mean the next release of 1.5 (in other words, the preview and stable releases), then no they're not scheduled to land in that time. If by next release you mean 1.6, then I can't say yet as those plans are still in progress. Thanks for checking!

**Comment by DHancock** [comment]
I may be being pedantic but there seems to be a problem with the package versions of the 1.5.0 experimental releases.

**1.5.0 exp 1** package version: **5000.1066.33.0**
**1.5.0 exp 2** package version: **5000.23.1950.0**

i.e. exp 2 is earlier than exp 1. Is that intensional or may be a cut and paste error?

**Reply by duncanmacmichael** [reply]
It was intentional, nothing to worry about. Thanks for noticing and asking, though!

**Comment by ChaimCL** [comment]
As we get closer to 1.5, is there any news of a community call from you? We're very concerned about that.You know, almost five months since the last one.
